install/README.md

```diff
@@ -120,6 +120,11 @@ of these settings or provide values to mandatory fields.
     - **Type:** `boolean`
     - **Default:** `false`
 
+- **`SS_AUDIT_LOG_MIDDLEWARE`**:
+    - **Description:** enable logging of authenticated users with responses.
```

Comment:
```suggestion
    - **Description:** enable X-Username header with authenticated HTTP responses.
```
Suggested text or similar? I feel like the context could be a little clearer. 

_(I don't know how this works as a second PR? - Just noticed it with the AM PR)_

Reply:
Nice suggestion! I'll make that change.

I'm not sure what you mean by `how this works as a second PR` - would you mind clarifying for me?

Reply:
Ah, gotcha! Thanks for clarifying

Reply:
I suspect I make this mistake a lot in conversation too. Too similar and used too often. Anyhoo!

Reply:
I'm sure I do too!